tuf/scripts/repo.py

```diff
@@ -592,7 +590,7 @@ def sign_role(parsed_arguments):
 
   for keypath in parsed_arguments.sign:
 
-    role_privatekey = import_privatekey_from_file(keypath)
+    role_privatekey = import_privatekey_from_file(keypath, parsed_arguments.pw)
```

Comment:
Doesn't this trigger the weirdo default behaviour where if --pw is not used, the string "pw" is used as password?

I guess I again don't really object changing one strange behaviour to another :shrug:

Reply:
Yes exactly. It's the case everywhere so I think it's better to have this strange behavior everywhere otherwise it's confusing